sound/hybrid_font.h

```diff
@@ -203,6 +203,7 @@ class HybridFont : public SaberBase, public Looper {
   RefPtr<BufferedWavPlayer> next_hum_player_;
   RefPtr<BufferedWavPlayer> swing_player_;
   RefPtr<BufferedWavPlayer> lock_player_;
+  RefPtr<BufferedWavPlayer> free_player_;
```

Comment:
I think this should be a local variable, not a class level variable.

Reply:
GetOrFreeWavPlayer needs to return it.
So it needs to be a local variable in multiple functions.

Reply:
ok figured it out